addons/web/controllers/main.py

```diff
@@ -1807,7 +1807,14 @@ def get_fields(self, model, prefix='', parent_name= '',
     def namelist(self, model, export_id):
         # TODO: namelist really has no reason to be in Python (although itertools.groupby helps)
         export = request.env['ir.exports'].browse([export_id]).read()[0]
-        export_fields_list = request.env['ir.exports.line'].browse(export['export_fields']).read()
+        exported_fields = request.env['ir.exports.line'].browse(export['export_fields'])
+
+        for invalid_fields in exported_fields.filtered(lambda f: f.name not in self.fields_get(model)):
```

Comment:
can you move the `self.fields_get(model)` outside of the lambda, in a variable, we don't want to call fields_get() on every field

Reply:
> can you move the `self.fields_get(model)` outside of the lambda, in a variable, we don't want to call fields_get() on every field

@Julien00859 , changes has been done.